modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -22,7 +32,9 @@ image::salt-ssh-contact-taigon.png[scaledwidth=80%]
 
 
 To use Push via Salt SSH, you must have the SSH daemon running on the client, and reachable by the [systemitem]``salt-api`` daemon running on the {productname} Server.
-Additionally, Python must be available on the remote system, and be a version supported by Salt.
+// FIXME: or salt-bundle
+Additionally, Python version supported by Salt must be available on the remote system.
```

Comment:
This is now wrong with SUMA 4.3. We always use salt-bundle with preflight script which brings its own python.

Reply:
ok